Detour/Source/DetourNavMesh.cpp

```diff
@@ -920,7 +920,7 @@ dtStatus dtNavMesh::addTile(unsigned char* data, int dataSize, int flags,
 	tile->detailVerts = (float*)d; d += detailVertsSize;
 	tile->detailTris = (unsigned char*)d; d += detailTrisSize;
 	tile->bvTree = (dtBVNode*)d; d += bvtreeSize;
-	tile->offMeshCons = (dtOffMeshConnection*)d; d += offMeshLinksSize;
+	tile->offMeshCons = (dtOffMeshConnection*)d; // d += offMeshLinksSize;
```

Comment:
IMO I think this dead-store makes things more clear to the reader that we're just reading data from a byte buffer and advancing a read pointer.  Additionally, it makes the code a bit more robust to changes, because you don't have to figure out if you need to un-comment some code that you didn't write.